app/src/main/kotlin/com/simplemobiletools/camera/views/CameraPreview.kt

```diff
@@ -39,13 +39,15 @@ import java.util.concurrent.TimeUnit
 // and video sample at https://github.com/googlesamples/android-Camera2Video
 class CameraPreview : ViewGroup, TextureView.SurfaceTextureListener, MyPreview {
 
+    private val TAG = "CameraPreview"
     private val FOCUS_TAG = "focus_tag"
     private val MAX_PREVIEW_WIDTH = 1920
     private val MAX_PREVIEW_HEIGHT = 1080
     private val MAX_VIDEO_WIDTH = 4096
     private val MAX_VIDEO_HEIGHT = 2160
     private val CLICK_MS = 250
     private val CLICK_DIST = 20
+    private val NUMBER_OF_SHOTS = 10 // the burst functionality will take 5 pictures
```

Comment:
You can remove this since it is unused; I believe this is from your previous implementation